build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -222,7 +222,7 @@ class RuntimeTestConfig {
         'report-config': {lcovonly: {file: `lcov-${testType}.info`}},
       };
 
-      const plugin = [
+      const istanbulPlugin = [
```

Comment:
This is just for our code coverage reports so no need to include the plugin here.